app/views/projects/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h3><%= @project.name %></h3>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
```

Comment:
This works, but I would consider this an MVC infraction since `.count` is a calculation - we can prove this by adding a pry and doing `@project.contestants.count.to_sql` and watching the constructed SQL statement change - and calculations should be done inside a model method and then tested.

Reply:
(To be clear, tested in a _model test_ specifically)